src/python/WMCore/MicroService/Unified/MSMonitor.py

```diff
@@ -17,123 +21,170 @@ class MSMonitor(MSCore):
     MSMonitor class provide whole logic behind
     the transferor monitoring module.
     """
+    def __init__(self, msConfig, logger=None):
+        super(MSMonitor, self).__init__(msConfig, logger)
+        # update interval is used to check records in CouchDB and update them
+        # after this interval, default 6h
+        self.updateInterval = self.msConfig.get('updateInterval', 6*60*60)
+        self.campaigns = {}
+
+    def updateCaches(self):
+        """
+        Fetch some data required for the monitoring logic, e.g.:
+         * all campaign configuration
+        :return: True if all of them succeeded, else False
+        """
+        campaigns = self.reqmgrAux.getCampaignConfig("ALL_DOCS")
+        if not campaigns:
+            self.logger.warning("Failed to fetch the campaign configurations")
+        else:
+            # let's make campaign look-up easier and more efficient
+            self.campaigns = {}
+            for camp in campaigns:
+                if 'campaignName' not in camp:
```

Comment:
`CampaignName`. Same comment for a few lines below

Reply:
Here I totally lost. In most of the code the convention is to use `lowerCase` instead of `CamelCase` notations. Why on earth you insist to use `CampaignName` instead of `campaignName`. To give you a few examples we used already `dataType`, `transferIDs`, `updateInterval`, `workflowName`. I think you're breaking your own rules and I don't really know where to use certain schema naming convention.

Please clarify?

Reply:
You're mixing up the document types. We're dealing mostly with two different documents in CouchDB:
* **campaign**: schema extensively discussed and agreed as in this issue: https://github.com/dmwm/WMCore/issues/9139#issue-430869532
* **transfers**: also discussed in lengthy and schema defined here: https://gist.github.com/amaltaro/72599f995b37a6e33566f3c749143154#file-transferor_format-json-L33

Most of the WMCore couchdb documents follow the capital camel case convention, like workload_cache and workqueue elements. So perhaps the transfer one should be adapted to follow the same as well...